include/web_video_server/h264_streamer.hpp

```diff
@@ -27,10 +27,12 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-#ifndef H264_STREAMERS_H_
-#define H264_STREAMERS_H_
+#ifndef WEB_VIDEO_SERVER__H264_STREAMER_HPP_
```

Comment:
I would not be opposed to switching to `#pragma once` at this point, but up to you!

Reply:
I agree, not a fan of header guards myself.